atomics/t1179/t1179.yaml

```diff
@@ -0,0 +1,18 @@
+attack_technique: T1179
+display_name: Hooking
+
+atomic_tests:
+- name: Hook PowerShell TLS Encrypt/Decrypt Messages
+  description: |
+    Hooks functions in PowerShell to read TLS Communications
+  supported_platforms:
+    - windows
+  input_arguments:
```

Comment:
see #161 I think we need a prerequisites step to define how to build the CPP into a DLL